package.json

```diff
@@ -65,6 +65,7 @@
     "@edx/browserslist-config": "^1.1.1",
     "@edx/frontend-build": "12.4.19",
     "@edx/reactifex": "2.1.1",
+    "@testing-library/react": "11.2.7",
```

Comment:
This is a commonly used package in our frontend projects that, in my opinion, make writing test a bit easier... so I added it along with my other changes.

Reply:
This is something I discussed with @jsnwesson. In some of the other MFEs, everything from `@testing-library/react` gets exported by the `setupTest.js` file. The Profile MFE doesn't have much in that file yet so I think it's _fine_ to import here. Although, when the SkillsBuilder eventually gets it's own MFE, we'll probably end up moving any imports from `@testing-library/react` to that `setupTest.js` file

Reply:
I had noticed this as well but figured this MFE is behind on what may be considered "standard configuration" these days. The benefits of having the test methods outweighed some of the extra work to bring it in line with more "modern" MFEs though.